tests/integration/tier0/latest-kernel-check/test_latest_kernel_check.py

```diff
@@ -46,3 +48,47 @@ def test_verify_latest_kernel_check_passes_with_failed_repoquery(shell, convert2
     if "centos-8.5" in get_system_release:
         assert shell(f"rm -f {centos_custom_reposdir}/centos-8.5/{repofile}.repo").returncode == 0
     assert shell(f"rm -f /etc/yum.repos.d/{repofile}.repo").returncode == 0
+
+
+@pytest.mark.yum_exclude_kernel
+def test_latest_kernel_check_with_exclude_kernel_option(shell, convert2rhel):
+    """
+    Define `exclude=kernel` in /etc/yum.conf and verify, the conversion is not inhibited with:
+    'CRITICAL - Could not find any kernel from repositories to compare against the loaded kernel.'
+    """
+
+    yum_config = "/etc/yum.conf"
+    backup_dir = "/tmp/config-backup"
+    config = configparser.ConfigParser()
+    config.read(yum_config)
+    exclude_option = r"exclude=.*kernel kernel-core\n"
+
+    assert shell(f"mkdir {backup_dir}").returncode == 0
+
+    assert shell(f"cp {yum_config} {backup_dir}").returncode == 0
+    # If there is already an `exclude` section, append to the existing value
+    if config.has_option("main", "exclude"):
+        pre_existing_value = config.get("main", "exclude")
+        config.set("main", "exclude", f"{pre_existing_value} kernel kernel-core")
```

Comment:
Maybe it would be too overkill, but do you think it's valid to check if the `pre_existing_value` is already containing `kernel` or `kernel-core`? I mean, imagining that the `exclude` option have a pre-existing value, like: 

```cfg
[main]
exclude = kernel
```

In that part of the test, we would end up having something like:

```cfg
[main]
exclude = kernel kernel kernel-core
```

Reply:
Reproducer: 

```
[root@c2r-20221117100038 ~]# cat /etc/yum.conf 
[main]
gpgcheck=1
installonly_limit=3
clean_requirements_on_remove=True
best=True
skip_if_unavailable=False
exclude=kernel-core kernel-core
```

```
[root@c2r-20221117100038 ~]# repoquery --quiet --qf "%{BUILDTIME}\t%{VERSION}-%{RELEASE}\t%{REPOID}" --setopt=reposdir=/usr/share/convert2rhel/repos/centos-8.5 kernel-core
[root@c2r-20221117100038 ~]# repoquery --setopt=exclude= --quiet --qf "%{BUILDTIME}\t%{VERSION}-%{RELEASE}\t%{REPOID}" --setopt=reposdir=/usr/share/convert2rhel/repos/centos-8.5 kernel-core
2021-10-19 15:34        4.18.0-348.el8  baseos
2021-11-16 15:02        4.18.0-348.2.1.el8_5    baseos
2021-12-22 13:45        4.18.0-348.7.1.el8_5    baseos
```

Reply:
Thanks for the investigation @r0x0d !